apps/antalmanac/src/components/Header/SettingsMenu.tsx

```diff
@@ -159,6 +177,18 @@ function ExperimentalMenu() {
                 </Box>
                 <Switch color="primary" value={previewMode} checked={previewMode} onChange={handlePreviewChange} />
             </Box>
+
+            <Box display="flex" justifyContent="space-between" width={1}>
+                <Box display="flex" alignItems="center" style={{ gap: 4 }}>
```

Comment:
Boxes can be styled either with the `style` prop, `sx` prop (MUIv5), or via the props directly. Generally, I prefer the former two (`style` and `sx`), since it's a consistent strategy for all components. For boxes, prefer not mixing them. i.e. You can use either, but just not at the same time.

```suggestion
                <Box style={{ display: 'flex', alignItems: 'center', gap: 4 }}>
```
Here, since gap was captured in a `style` prop, just move all the other styles to the `style` prop as well. Or vice versa if that's possible and preferrable.